packages/terra-site/postcss.config.js

```diff
@@ -0,0 +1,23 @@
+const Autoprefixer = require('autoprefixer');
+const CustomProperties = require('postcss-custom-properties');
+const rtl = require('postcss-rtl');
+const ThemingPlugin = require('./theming-plugin');
+
+module.exports = {
+  plugins() {
+    return [
+      Autoprefixer({
+        browsers: [
+          'ie >= 10',
+          'last 2 versions',
+          'last 2 android versions',
+          'last 2 and_chr versions',
+          'iOS >= 8',
```

Comment:
should this be bumped to 10?

Reply:
Yes, our currently minimum supported version is 10.2